### PR TITLE
[EDU-3193] feat: add activate dnssec

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page on how to activate DNSSEC via API.

[EDU-3193]: https://aziontech.atlassian.net/browse/EDU-3193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+namespace: docs_secure_activate_dnssec
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/activate-dnssec/
+---
+
+Domain Name System Security Extensions (DNSSEC) provides an extra layer of security to verify the authenticity and integrity of a IP address response. Azion provides [DNSSEC compatibility](/en/documentation/products/intelligent-dns/dnssec-compatibility/) as long as your top-level domain (TLD) registry supports it and [your zone is configured](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/) with DNSSEC-related resource records on [Intelligent DNS](/en/documentation/products/guides/secure/add-records/).
+
+## Activate DNSSEC via API
+
+DNSSEC activation is available via [Azion API](https://api.azion.com/#f0f730a3-9739-40cf-ad07-0b7bca8abac8):
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:14]:
```suggestion
## Activate DNSSEC 

### Via API

DNSSEC activation is available via [Azion API](https://api.azion.com/#f0f730a3-9739-40cf-ad07-0b7bca8abac8):
```

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      "name": "A hosted zone",
+      "id": 1234
+    }
+  ],
+  "schema_version": 3
+}
+```
+
+3. Copy the `<id>` value of the specific zone in which you want to activate DNSSEC. In this example, it's `1234`.
+4.  Run the following `PATCH` request in your terminal, replacing [TOKEN VALUE] with your [personal token](/en/documentation/products/guides/personal-tokens/):
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:47]:
```suggestion
4. Run the following `PATCH` request in your terminal, replacing [TOKEN VALUE] with your [personal token](/en/documentation/products/guides/personal-tokens/):
```

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+{
+  "results": {
+    "is_enabled": true,
+    "status": "waiting"
+  },
+  "schema_version": 3
+}
+```
+
+Now, DNSSEC is active in the Intelligent DNS zone you specified.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:71]:
```suggestion
Now, DNSSEC is active in the Intelligent DNS zone you specified.

---

import ContributorList from '~/components/ContributorList.astro'

**Contributors** <ContributorList>Contributor</ContributorList>
```

Comment (bru-andrade) [reply src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:71]: Faltou o check the Azion API também

**Review by bru-andrade — APPROVED** [review]
Alguns comentários!

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+title: Activate DNSSEC
+description: Make use of DNSSEC compatibility to guarantee the authenticity and integrity of a IP address response.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:3]:
```suggestion
description: Use DNSSEC compatibility to guarantee the authenticity and integrity of an IP address response.
```

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+meta_tags: edge, secure, dnssec, dns
+namespace: docs_secure_activate_dnssec
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/activate-dnssec/
+---
+
+Domain Name System Security Extensions (DNSSEC) provides an extra layer of security to verify the authenticity and integrity of a IP address response. Azion provides [DNSSEC compatibility](/en/documentation/products/intelligent-dns/dnssec-compatibility/) as long as your top-level domain (TLD) registry supports it and [your zone is configured](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/) with DNSSEC-related resource records on [Intelligent DNS](/en/documentation/products/guides/secure/add-records/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:10]:
```suggestion
**Domain Name System Security Extensions (DNSSEC)** provides an extra layer of security to verify the authenticity and integrity of an IP address response. Azion provides [DNSSEC compatibility](/en/documentation/products/intelligent-dns/dnssec-compatibility/) as long as your top-level domain (TLD) registry supports it and [your zone is configured](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/) with DNSSEC-related resource records on [Intelligent DNS](/en/documentation/products/guides/secure/add-records/).
```

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## Activate DNSSEC via API
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:12]:
```suggestion

---

## Activate DNSSEC via API
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx (hunk 5 above)
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:10]:
```suggestion
Domain Name System Security Extensions (DNSSEC) provide an extra layer of security to verify the authenticity and integrity of an IP address response. Azion provides [DNSSEC compatibility](/en/documentation/products/intelligent-dns/dnssec-compatibility/) as long as your top-level domain (TLD) registry supports it and [your zone is configured](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/) with DNSSEC-related resource records on [Intelligent DNS](/en/documentation/products/guides/secure/add-records/).
```

Comment (gabriel-azion) [reply src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:10]: the `extensions` made me suggest that

File: src/content/docs/en/pages/secure-journey/activate-dnssec.mdx (hunk 3 above)
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/activate-dnssec.mdx:71]:
```suggestion
Now, DNSSEC is active in the Intelligent DNS zone you specified.

:::tip
Check the [Azion API documentation](https://api.azion.com/) and the [OpenAPI specification](https://github.com/aziontech/azionapi-openapi/) to know more about all features available via API.
:::

```

**Pull request merged**